docs/examples/error_handler/error_handler_0/src/error_handler_0/version.py

```diff
@@ -12,4 +12,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-__version__ = "0.23.dev0"
+__version__ = "0.23b0"
```

Comment:
not sure I understand why this is needed here

Reply:
Fixed